Tests/SentryTests/Integrations/SentryCrash/SentryCrashIntegrationTests.swift

```diff
@@ -52,22 +47,22 @@ class SentryCrashIntegrationTests: NotificationCenterTestCase {
         }
     }
     
-    private let fixture = Fixture()
+    private lazy var fixture = Fixture()
     
     override func setUp() {
         super.setUp()
-        CurrentDate.setCurrentDateProvider(fixture.currentDateProvider)
+        CurrentDate.setCurrentDateProvider(TestCurrentDateProvider())
         
-        fixture.fileManager.deleteCurrentSession()
-        fixture.fileManager.deleteCrashedSession()
-        fixture.fileManager.deleteAppState()
+        fixture.client.fileManager.deleteCurrentSession()
+        fixture.client.fileManager.deleteCrashedSession()
+        fixture.client.fileManager.deleteAppState()
     }
     
     override func tearDown() {
         super.tearDown()
-        fixture.fileManager.deleteCurrentSession()
-        fixture.fileManager.deleteCrashedSession()
-        fixture.fileManager.deleteAppState()
+        fixture.client.fileManager.deleteCurrentSession()
+        fixture.client.fileManager.deleteCrashedSession()
+        fixture.client.fileManager.deleteAppState()
```

Comment:
These are going to change in the subsequent PR dealing more with test state: https://github.com/getsentry/sentry-cocoa/pull/2426